github-management/OWNERS

```diff
@@ -4,11 +4,11 @@ approvers:
   - cblecker
   - mrbobbytables
   - nikhita
+  - palnabarun
+  - MadhavJivrajani
 
 reviewers:
```

Comment:
not having the approvers on the reviewers list means only ihor will be assigned reviews which doesn't seem likely to be the intention currently?